packages/node_modules/@ciscospark/widget-meet/src/container.js

```diff
@@ -34,7 +37,7 @@ export class MeetWidget extends Component {
 
   shouldComponentUpdate(nextProps) {
     const props = this.props;
-    return nextProps.media !== props.media;
+    return nextProps.media !== props.media || nextProps.toPersonAvatar !== props.toPersonAvatar;
```

Comment:
Sometimes the avatars wouldn't have loaded and we wouldn't get an update once they did.